tutorials/hist/Fibonacci.C

```diff
@@ -30,7 +35,7 @@ void Fibonacci(int N=7) {
    C->SetFrameLineWidth(0);
 
    TH2Poly *h2pf = new TH2Poly(); // TH2Poly containing Fibonacci bins.
-   h2pf->SetTitle(Form("The first %d Fibonacci numbers",N));
+   h2pf->SetTitle(Form("The first %i Fibonacci numbers",N));
```

Comment:
This routine follows the formatting rules described at:
```
http://www.cplusplus.com/reference/cstdio/printf/
```
`%d` and `%i` are equivalent.
The ROOT code uses `%d` for integer in all other places.

Reply:
Ah, I see